vic/drivers/cesm/src/vic_cesm_init_library.c

```diff
@@ -88,37 +88,37 @@ initialize_l2x_data(void)
 
     size_t                  i;
 
-    log_info("Setting all l2x fields to %f", SHR_CONST_SPVAL);
+    log_info("Initializing l2x_data_struct");
 
     for (i = 0; i < local_domain.ncells_active; i++) {
         l2x_vic[i].l2x_Sl_t = SHR_CONST_SPVAL;
-        l2x_vic[i].l2x_Sl_tref = SHR_CONST_SPVAL;
-        l2x_vic[i].l2x_Sl_qref = SHR_CONST_SPVAL;
+        l2x_vic[i].l2x_Sl_tref = 0;
+        l2x_vic[i].l2x_Sl_qref = 0;
         l2x_vic[i].l2x_Sl_avsdr = SHR_CONST_SPVAL;
         l2x_vic[i].l2x_Sl_anidr = SHR_CONST_SPVAL;
         l2x_vic[i].l2x_Sl_avsdf = SHR_CONST_SPVAL;
         l2x_vic[i].l2x_Sl_anidf = SHR_CONST_SPVAL;
-        l2x_vic[i].l2x_Sl_snowh = SHR_CONST_SPVAL;
-        l2x_vic[i].l2x_Sl_u10 = SHR_CONST_SPVAL;
-        l2x_vic[i].l2x_Sl_ddvel = SHR_CONST_SPVAL;
-        l2x_vic[i].l2x_Sl_fv = SHR_CONST_SPVAL;
-        l2x_vic[i].l2x_Sl_ram1 = SHR_CONST_SPVAL;
-        l2x_vic[i].l2x_Sl_logz0 = SHR_CONST_SPVAL;
-        l2x_vic[i].l2x_Fall_taux = SHR_CONST_SPVAL;
-        l2x_vic[i].l2x_Fall_tauy = SHR_CONST_SPVAL;
-        l2x_vic[i].l2x_Fall_lat = SHR_CONST_SPVAL;
-        l2x_vic[i].l2x_Fall_sen = SHR_CONST_SPVAL;
+        l2x_vic[i].l2x_Sl_snowh = 0;
+        l2x_vic[i].l2x_Sl_u10 = 0;
+        l2x_vic[i].l2x_Sl_ddvel = 0;
+        l2x_vic[i].l2x_Sl_fv = 0;
+        l2x_vic[i].l2x_Sl_ram1 = 0;
+        l2x_vic[i].l2x_Sl_logz0 = 0;
+        l2x_vic[i].l2x_Fall_taux = 0;
+        l2x_vic[i].l2x_Fall_tauy = 0;
+        l2x_vic[i].l2x_Fall_lat = 0;
+        l2x_vic[i].l2x_Fall_sen = 0;
         l2x_vic[i].l2x_Fall_lwup = SHR_CONST_SPVAL;
-        l2x_vic[i].l2x_Fall_evap = SHR_CONST_SPVAL;
-        l2x_vic[i].l2x_Fall_swnet = SHR_CONST_SPVAL;
-        l2x_vic[i].l2x_Fall_fco2_lnd = SHR_CONST_SPVAL;
-        l2x_vic[i].l2x_Fall_flxdst1 = SHR_CONST_SPVAL;
-        l2x_vic[i].l2x_Fall_flxdst2 = SHR_CONST_SPVAL;
-        l2x_vic[i].l2x_Fall_flxdst3 = SHR_CONST_SPVAL;
-        l2x_vic[i].l2x_Fall_flxdst4 = SHR_CONST_SPVAL;
-        l2x_vic[i].l2x_Fall_flxvoc = SHR_CONST_SPVAL;
-        l2x_vic[i].l2x_Flrl_rofliq = SHR_CONST_SPVAL;
-        l2x_vic[i].l2x_Flrl_rofice = SHR_CONST_SPVAL;
+        l2x_vic[i].l2x_Fall_evap = 0;
+        l2x_vic[i].l2x_Fall_swnet = 0;
+        l2x_vic[i].l2x_Fall_fco2_lnd = 0;
+        l2x_vic[i].l2x_Fall_flxdst1 = 0;
+        l2x_vic[i].l2x_Fall_flxdst2 = 0;
+        l2x_vic[i].l2x_Fall_flxdst3 = 0;
+        l2x_vic[i].l2x_Fall_flxdst4 = 0;
+        l2x_vic[i].l2x_Fall_flxvoc = 0;
+        l2x_vic[i].l2x_Flrl_rofliq = 0;
+        l2x_vic[i].l2x_Flrl_rofice = 0;
```

Comment:
I've gone back to setting all to `SHR_CONST_SPVAL` for now. But I think we should revisit this, as we set the coupler fields to 0 previously for VIC in RASM (I'll make an issue for it so we can figure out what we want to do).